src/board/pagelistener.tsx

```diff
@@ -5,13 +5,7 @@ import { KonvaEventObject } from "konva/types/Node"
 import { Stage } from "konva/types/Stage"
 import { Vector2d } from "konva/types/types"
 import store from "../redux/store"
-import {
-    startLiveStroke,
-    moveLiveStroke,
-    registerLiveStroke,
-    getPageIndex,
-    abortLiveStroke,
-} from "../drawing/strokeactions"
+import * as actions from "../drawing/stroke/actions"
```

Comment:
I like destructuring more, but this is fine also

Reply:
I'd like to propose to use named imports in the future for better readability. What do you think?